fe/fe-core/src/main/java/com/starrocks/scheduler/Task.java

```diff
@@ -0,0 +1,117 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.scheduler;
+
+import com.google.gson.annotations.SerializedName;
+import com.starrocks.common.io.Text;
+import com.starrocks.common.io.Writable;
+import com.starrocks.persist.gson.GsonUtils;
+import com.starrocks.statistic.Constants;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.Map;
+
+public class Task implements Writable {
+
+    @SerializedName("id")
+    private Long id;
+
+    @SerializedName("name")
+    private String name;
+
+    @SerializedName("taskType")
+    private Constants.TaskType taskType = Constants.TaskType.MANUAL;
+
+    @SerializedName("createTime")
+    private LocalDateTime createTime;
```

Comment:
Is json serialization supported for `LocalDateTime` type?
Why not use long type timestamp?

Reply:
We have the LocalDateTime adapter in the GSONUtils

Reply:
You are right.

Reply:
I think use long type timestamp is a better way.